.travis.yml

```diff
@@ -8,26 +8,32 @@ php:
   - 7.0
   - hhvm
 
-matrix:
-  allow_failures:
-    - php: 7.0
-    - php: hhvm
+sudo: false
 
 cache:
   directories:
     - $HOME/.composer/cache
+    - vendor
```

Comment:
I don't get the point of caching vendors and running a `composer update`. The cache’s purpose is to make installing language-specific dependencies easy and fast, but our concern is to test the codebase against many changing vendors dependencies.

This also has an unresolved side effect over the `Makefile`. 

In addition, & I don't want to be asked to manually clear the Travis cache every time a PR encounter a BC break issue and the PR author wants to digg into a reliable travis job trace.

For now, I'd suggest not to cache the vendors, unless we found a reason to do so.
I'm open to any clear demonstration about an opposite POV.